docs/docs/guides/tools/inference.md

```diff
@@ -329,7 +329,7 @@ Once you run the code sample, you can view the trace in Weave by clicking the li
 
 Next, try the [advanced example](#advanced-example-use-weave-evaluations-and-leaderboards-with-the-inference-service).
 
-![Traces display](imgs/image.png)
+![Traces display](./imgs/image.png)
```

Comment:
Why is it doing this? Several more below.

Reply:
Page removed